lcls-twincat-motion/Library/POUs/Motion/FB_MotionStage.TcPOU

```diff
@@ -93,7 +105,7 @@ END_CASE
 // Handle the brake. TRUE means brake disabled.
```

Comment:
Mostly unrelated to this PR, but if `.bBrake` `True` means disabled, think we could rename it to `.bBrakeDisengaged` or disabled... or something even less confusing?

Reply:
I agree, but I couldn't come up with a succinct name. `True` is disabled because all of our breaks are NC.

Reply:
Pedantic, but I think it's helpful to really clarify disabled == disengaged.

Reply:
Should I approach this by adding a huge comment, or by finding an appropriate rename?

Reply:
My opinion: I think an appropriate rename would be advisable, or confusion in boolean state could easily lead to bugs down the line.

Reply:
Some options, keeping in mind that this is a hardware output variable:
```
bDisableBrake
bBrakeDisable
bNoBrake
bUnBrake
bReleaseBrake
bBrakeRelease
bBrakeOff
```

Reply:
I'll second @teddyrendahl .... `bBrakeRelease`